app/Services/MapboxGeocodingService.php

```diff
@@ -26,4 +29,35 @@ public function getCoordinatesFromApi(string $cityName, string $countryName): ar
             return [];
         }
     }
+
+    public function getPlaceFromApi(string $lat, string $long): array
+    {
+        $token = env("MAPBOX_TOKEN");
+        $client = new Client();
+
+        try {
+            $response = $client->get(
+                "https://api.mapbox.com/geocoding/v5/mapbox.places/$long,$lat.json?access_token=$token",
```

Comment:
I would like to move API URL to some variable or even to the config. It is used twice in this class.

Reply:
moved to config